charts/substra-backend/values.yaml

```
@@ -367,6 +367,24 @@ worker:
         memory: "1Gi"
       limits:
         memory: "64Gi"
+    ## @param worker.computePod.nodeSelector Node labels for pod assignment
+    ##
+    nodeSelector: {}
+    ## @param worker.computePod.tolerations Toleration labels for pod assignment
+    ##
+    tolerations: []
+    ## @param worker.computePod.affinity Worker compute pod container affinity.
+    ##
+    affinity:
+      podAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+          matchExpressions:
+          - key: statefulset.kubernetes.io/pod-name
+            operator: In
+            values:
+            - $(HOSTNAME)
```

Incorrect indentation

```suggestion
            matchExpressions:
            - key: statefulset.kubernetes.io/pod-name
              operator: In
              values:
              - $(POD_NAME)
```